frontend/src/views/SettingsView.vue

```diff
@@ -20,9 +20,8 @@ const { t } = useI18n({ useScope: 'global' });
 const route = useRoute();
 const router = useRouter();
 const sections = enumToObject(SettingsSections);
-const routeView = route.params.view as string;
-
-const activeView = computed<number>(() => (routeView && sections[routeView] ? sections[routeView] : SettingsSections.General));
+// Note: Use direct variables in computed, otherwise it won't be updated if transformed (like by typing)
+const activeView = computed<number>(() => (route.params.view && sections[route.params.view as string] ? sections[route.params.view as string] : SettingsSections.General));
```

Comment:
Noticed after my merge that the settings menu was broken. Since VueJS router seems to mistype route.params.view (or is it typescripts fault?) I see why we cast it as a string into a variable, but by doing that it breaks reactivity. (Unless we wrap it in a ref, which meh)

Reply:
Yes this is so annoying. Didn't know this was breaking reactivity 😱 I guess the problem is, that the default route param type includes string[] as well. We might want to take a look at https://router.vuejs.org/guide/advanced/typed-routes.html, but they say its experimental?? I mean, surely we're not the only ones with that issue 😅